src/screens/ProfileScreen/index.js

```diff
@@ -197,6 +199,8 @@ const ProfileScreen = ({route}) => {
     reloadFetchAnonymousPost,
     getProfileCache
   } = useProfileScreenHook();
+
+  const {fetchNextFeeds} = useFeedPreloadHook(mainFeeds?.length, () => getMyFeeds(postOffset));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useFeedPreloadHook` has been added. Ensure that its return value `fetchNextFeeds` is used correctly and the hook is implemented properly.
```diff
+ const {fetchNextFeeds} = useFeedPreloadHook(mainFeeds?.length, () => getMyFeeds(postOffset));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->